Metaverse/Makefile

```diff
@@ -70,3 +70,20 @@ ifeq ($(UNAME), Darwin)
 		-cook -build -stage -prereqs -pak -package -archive \
 		-archivedirectory=$(PWD)/archive
 endif
+
+cmake:
+ifeq ($(UNAME), Darwin)
+	/Users/Shared/Epic\ Games/UE_5.1/Engine/Build/BatchFiles/Mac/GenerateProjectFiles.sh \
```

Comment:
Are these paths constant or is there a possibility that it can create conflicts on other machines?

Reply:
Yes, they are Constant.

Reply:
ok lgtm